src/stringnumber.ts

```diff
@@ -27,14 +27,12 @@ export class StringNumber extends Modifier {
   }
 
   static get metrics(): StringNumberMetrics {
-    return (
-      Tables.currentMusicFont().getMetrics().stringNumber ?? {
-        verticalPadding: 0,
-        stemPadding: 0,
-        leftPadding: 0,
-        rightPadding: 0,
-      }
-    );
+    return Tables.lookupMetric('StringNumber', {
+      verticalPadding: 0,
+      stemPadding: 0,
+      leftPadding: 0,
+      rightPadding: 0,
+    });
```

Comment:
Do we need to pass in a default value here? tables.ts already has a set of values for StringNumber.

Reply:
reworked